src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationFeedbackPresenter.cs

```diff
@@ -165,6 +167,57 @@ private void updateFeedback(ParameterIdentificationRunState state)
          _activeFeedbackPresenter?.UpdateFeedback(state);
       }
 
+      public void OnFormClosed()
+      {
+         //Nothing to do here
+      }
+
+      public void Close()
+      {
+         Handle(new ProjectClosedEvent());
+      }
+
+      public IPresentationSettings GetSettings()
+      {
+         //Nothing to do here
+         return null;
+      }
+
+      public void RestoreSettings(IPresentationSettings settings)
+      {
+         //Nothing to do here
+      }
+
+      public void SaveChanges()
+      {
+         //Nothing to do here
+      }
+
+      public void Activated()
+      {
+         //Nothing to do here
+      }
+
+      public void Handle(RenamedEvent eventToHandle)
+      {
+         //Nothing to do here
+      }
+
+      public void Edit(ParameterIdentificationFeedback objectToEdit)
+      {
+         _key = objectToEdit;
+         if (objectToEdit.Running)
+            Handle(new ParameterIdentificationStartedEvent(objectToEdit.ParameterIdentification));
```

Comment:
This is also very weird.  THis is not how the publisher works. Want to have a call?